### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by brunotl** [jpbarrette/curlpp#170]
the latest official release is 0.8.1 (2017), a new one is needed to ask to vcpkg maintainer to upgrade it's port ...

**Comment by dg0yt** [comment]
Vcpkg port `curlpp` uses scheme `version-date`, so it is not bound to releases.

And it won't build with tomorrow's curl releases due to https://github.com/jpbarrette/curlpp/issues/169.

**Comment by chenrui333** [comment]
cc @jpbarrette @sgallou can you help with cutting a new release? Thanks!

**Comment by jpbarrette** [comment]
I'll try to get that done this weekend

**Comment by jnordin20** [comment]
Any updates on the next release?

**Comment by rosset** [comment]
hi, maybe it's time for a new release ?

=> the master branch is 75 commits ahead of 0.8.1
=> there are important fixes there specially for distro package maintainers

https://github.com/jpbarrette/curlpp/compare/v0.8.1...master

**Comment by omatasas** [comment]
Any updates on the next release?

**Comment by camelback** [comment]
> hi, maybe it's time for a new release ?
> 
> => the master branch is 75 commits ahead of 0.8.1 => there are important fixes there specially for distro package maintainers
> 
> [v0.8.1...master](https://github.com/jpbarrette/curlpp/compare/v0.8.1...master)

I agree with @rosset, there should be a new "official" release. One could argue that the conan-center recipe `0.8.1.cci.20240530` is "unofficial", and should be `0.9.x.*`

**Comment by k0ste** [comment]
7 years since last tag. Cmake 4 now. Can you tag for new tarball please?
